libr/io/map.c

```diff
@@ -26,9 +26,12 @@ static int _cmp_map_event(const void *a_, const void *b_) {
 	struct map_event_t *a = (void *)a_, *b = (void *)b_;
 	ut64 addr0 = a->addr - a->is_to, addr1 = b->addr - b->is_to;
 	if (addr0 != addr1) {
-		return addr0 < addr1 ? -1 : 1;
+		return addr0 < addr1? -1: 1;
 	}
-	return a->is_to - b->is_to;
```

Comment:
what about return -1, 0 or 1 instead of abusing the integer overflows here

Reply:
ok